gcp-vertex-ai-mojo-scorer/README.md

```diff
@@ -0,0 +1,112 @@
+# DAI Deployment Template for GCP Vertex AI
+
+The docker image that is built by this project can be pushed to gcr.io and used for scoring
+Driverless AI Mojos in GCP Vertex AI. 
+
+## Building
+
+Make sure you are in the working directory `dai-deployment-templates`. Typing `pwd` in the terminal
+shell should have a similar output to `/my/path/to/dai-deployment-templates`.
+
+You can build the Docker image in two ways. The first only includes the ability to score and the second
```

Comment:
I would maybe be more explicit as to what are the ramifications of this. I.e. different images (and which ones), that python preprocessing requires the java to run python code, etc.